app/utils/report/test.py

```diff
@@ -227,7 +227,18 @@ def create_test_report(data, email_format, db_options,
         "test_groups": groups,
     }
 
-    template = plan_options.get("template", "test.txt")
+    if test_template:
+        path = os.getcwd() + "/utils/report/templates"
+        templates = [t for t in os.listdir( path )]
+        test_template = str(test_template) + ".txt"
+        if test_template in templates:
+            template = plan_options.get("template", test_template)
+        else:
+            utils.LOG.warning("This template does not exist")
```

Comment:
In the warning log, please add the filename of the template that it was looking for.